wordpress-mobile/MediaPicker-iOS
# Implement a Media Picker Input View

This PR implements a view integrating a mediaPicker that can be used as an inputView.

How to test:
 - Launch the demo app
 - Tap on the input field to on the top
 - Check if custom input view shows up
 - Select media on it
 - Tap Done
 - See if media shows up in the list
 - Play around with the media options to see if the scenarios work ok.
 - Push it to the limit.
